tests/cli/test_alice.py

```diff
@@ -7,6 +8,15 @@
 from nucypher.cli.actions import SUCCESSFUL_DESTRUCTION
 
 
+@mock.patch('nucypher.config.characters.AliceConfiguration.default_filepath', return_value='/non/existent/file')
```

Comment:
Nice - Inspired by the recent mocking work in birdcage? :-)

Reply:
😉